gno.land/cmd/gnoland/testdata/addpkg_invalid.txtar

```diff
@@ -0,0 +1,21 @@
+# test for add package; ensuring type checker catches invalid code.
+
+# start a new node
+gnoland start
+
+# add bar package located in $WORK directory as gno.land/r/foobar/bar
+! gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/foobar/bar -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
```

Comment:
What do you think about using `loadkpkg` for this?

Reply:
actually, we specifically want addpkg here!

we want to see what's the node response when adding a bad package. if we used loadpkg, then the package would be added at genesis, and then we wouldn't be able to see the error message.

Reply:
Ah 🤦‍♂️ 

Thank you for clarifying 🙏